### PR TITLE
Change for cuda verify integration

### PR DESCRIPTION
**Pull request by sakridge**

### CODE REVIEW

File: src/event.rs [hunk 1]
```diff
@@ -47,7 +47,7 @@ impl Event {
     /// spending plan is valid.
     pub fn verify(&self) -> bool {
         match *self {
-            Event::Transaction(ref tr) => tr.verify(),
+            Event::Transaction(ref tr) => tr.verify_plan(),
```
Comment (garious) [inline src/event.rs]: This change shouldn't be necessary. Can you revert that and report back any consequences.

File: src/transaction.rs [hunk 2]
```diff
@@ -8,12 +8,17 @@ use rayon::prelude::*;
 use signature::{KeyPair, KeyPairUtil, PublicKey, Signature, SignatureUtil};
 
 #[derive(Serialize, Deserialize, Debug, PartialEq, Eq, Clone)]
-pub struct Transaction {
+pub struct Signed {
```
Comment (garious) [inline src/transaction.rs]: Need to rename this to TransactionData

File: src/transaction.rs [hunk 3]
```diff
@@ -8,12 +8,17 @@ use rayon::prelude::*;
...
     pub from: PublicKey,
```
Comment (garious) [inline src/transaction.rs]: Need to leave `from` in the top-level struct

File: src/event.rs (hunk 1 above)
Comment (sakridge) [reply src/event.rs]: I changed to verify_sig(), but did you mean it should be verify_sig() && verify_plan(). Anatoly removed the one that does both, but I can add it back.

**Comment by sakridge** [comment]
I didn't include the GPU lib, do we want to just check it in for now?

**Comment by garious** [comment]
The functionality tucked under feature flags don't need to be buildable by CI. This PR is good to go. Merging.

**Pull request merged**